roles/openshift-prep/tasks/prerequisites.yml

```diff
@@ -6,19 +6,14 @@
   yum:
     name: "{{ item }}"
     state: latest
-  with_items:
-    - wget
-    - git
-    - net-tools
-    - bind-utils
-    - bridge-utils
-    - bash-completion
-    - vim-enhanced
+  with_items: "{{ required_packages }}"
+  when: manage_packages|bool
 
 - name: "Update all packages (this can take a very long time)"
   yum:
-    name: "*"
+    name: '"*"'
```

Comment:
@bogdando can you revert this line? I'm seeing this error with it:

```
fatal: [app-node-0.openshift.example.com]: FAILED! => {"changed": false, "failed": true, "msg": "No package matching '\"*\"' found available, installed or updated", "rc": 126, "results": ["No package matching '\"*\"' found available, installed or updated"]}
```

And the example in the yum module docs shows a single set of quotes:

http://docs.ansible.com/ansible/yum_module.html